### ISSUE TITLE
Could not change URL on Hass Gestalt

### ISSUE THREAD

**Issue by plugfir3** [BimmerGestalt/IDriveConnectAddons#10]
I would like to change url in Hass Gestalt; however, I couldn't delete or change the existing url that I put in before. I pressed the logout button and nothing happened. Even I reinstalled the app, the existing url is still there. Any suggestions?

**Comment by hufman** [comment]
I don't believe the code ever changes the field to be uneditable, you should be able to just start typing in it. Are you finding this to not be the case?

**Comment by plugfir3** [comment]
Yes, I couldn't delete or add any letter to the field.

https://user-images.githubusercontent.com/33976265/173198327-0ffa1022-9882-4305-9dd4-ccd70a3c1a27.mp4

**Comment by hufman** [comment]
I was able to reproduce this, it'll take a bit of thinking to rearrange the Oauth layer to fix it. Thank you for reporting!
You should be able to wipe the app's storage to get it back to a blank slate to work around this for now.
